data/remnant missions.txt

```diff
@@ -495,3 +565,691 @@ mission "Remnant: Technology Available"
+			choice
+				`	"I could do that."`
+				`	"I'm not interested."`
+					decline
+			`	Tali quickly helps load the cargo onboard your ship. It doesn't appear to be actual whole items anymore, but rather pieces thereof. "They will be expecting you, but you might have to pry them out of their lab to get some help unloading. They can be a bit single-minded when they have something interesting to work on."`
+				accept
+	
+	on complete
+		payment 150000
+		dialog `On Caelian, you are able to find the appropriate research warehouse just off the main landing pad. After a few minutes, a lab technician disentangles herself from a project and brings a wagon and camel team out to the ship to unload the salvage. It strikes you as a bit incongruous that a lab technician is using a graviton repulsion lifter to load cargo from your ship into a wagon pulled by a camel. "If you can, I should have some things to send back to Tali if you could meet me in the cafeteria this afternoon."`
+
+
+
+mission "Remnant: Salvage 6"
+	name "Transport outfits to Tali"
+	description `Deliver <cargo> to Viminal.`
+	source "Caelian"
+	cargo "ship parts" 20
+	blocked `The technician you delivered Tali's salvaged parts to would like you to take some finished parts back to Tali, but you will need 20 tons of space to do so.`
+	destination "Viminal"
+	to offer
+		has "Remnant: Salvage 5: done"
+	on offer
+		event "Remnant Salvage Available"
+		conversation
+			`You spot the laboratory technician entering the cafeteria, and she approaches you.`
+			`	"Thank you for waiting. I have a shipment of parts ready for Tali, and with the recent increase in attacks she needs them sooner rather than later. Please take them to her."`
+			`	Outside you find a camel pulling a wagon filled with ship parts to be loaded on board <ship>.`
+				accept
+		
+	on complete
+		payment 150000
+		conversation 
+			`Tali meets you at the dock with a flatbed ready to offload the shipment. As you run through your shutdown routine you hit the switch for the cargo bay doors, and by the time you have gone down to the hold, Tali has half the shipment already unloaded.`
+			`	Her signs flick at you with a gentleness that you are slowly deciding equates to a cheerful tone. "Thank you for all the help you've provided in recovering this salvage," she says as she hands you <payment>. "I have discussed it with the other engineers, and we have decided to give you access to our salvage."`
+			`	She leads you over to the warehouse, scans her ID on the console, and taps a few commands. Then she scans your ID card. "There you go, I have updated your access codes," she signs cheerfully. She hands you your ID and pulls you to a door on the far side of the outfitting shop. Beyond the door is a room filled with outfits and weapons that clearly don't fit the same style as the other Remnant technology. You have seen most of the things here on scans and when salvaging equipment from other ships, but never up close.`
```

Comment:
Anything holding these kinds of things en masse is only technically a `room`. I thought of a cubbyhole before reading the next 2 lines.